core/templates/cowdata.h

```diff
@@ -192,7 +192,14 @@ class CowData {
 	_FORCE_INLINE_ void set(Size p_index, const T &p_elem) {
 		ERR_FAIL_INDEX(p_index, size());
 		_copy_on_write();
+#if defined(__GNUC__) && !defined(__clang__) // False positive raised when using constexpr.
```

Comment:
Maybe this should be moved to the top of the file, to the other diagnostics?
Also, you have those nice wrappers now =)

Reply:
This isolated it to the one bit of code that acted up, but given there's now already a warning at the top I suppose there isn't any harm.